packages/google-closure-compiler-windows/build-image.js

```diff
@@ -24,7 +24,7 @@
 const fs = require('fs');
 const path = require('path');
 const {DIM, RESET} = require('../../build-scripts/colors');
-const runCommand = require('../../build-scripts/run-command');
+const {spawn} = require('child_process');
```

Comment:
This seems like an unrelated fixup.

Reply:
Yeah just missed when I split the github actions PR out.